website/server/models/task.js

```diff
@@ -93,6 +101,15 @@ TaskSchema.statics.sanitizeReminder = function sanitizeReminder (reminderObj) {
   return reminderObj;
 };
 
+TaskSchema.statics.checkShortNameAvailability = async function checkShortNameAvailability (userId, shortName) {
+  let taskWithSameShortname = await this.findOne({
+    userId,
+    shortName,
+  }).exec();
```

Comment:
here you should only fetch the task `_id` (only because you cannot fetch no data)